provisioning/miniprov/hare_mp/main.py

```diff
@@ -375,8 +376,8 @@ def prepare(args):
     stop_consul_blocking(consul_starter)
 
 
-def get_hare_motr_s3_processes(utils: ConsulUtil) -> Dict[str, List[Fid]]:
-    nodes = utils.catalog.get_node_names()
+def get_hare_motr_s3_processes(utils: ConfigManager) -> Dict[str, List[Fid]]:
```

Comment:
I think, having ConfigManager and Utils separate is better. Utils can take ConfigManager as an initialization parameter and internally can invoke corresponding kv_get() apis.
With Utils as a separate module, we can move some common functions to utils and keep ConfigManager specific implementation (e.g. Consul) separate.